setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup
+
+test_dependencies = ['pytest']
+
+setup(
+   name='brdnet',
+   version='1.0',
+   description='Transfer Learning on Gene Expression Data',
+   author='Ben Heil',
+   author_email='foomail@foo.com',
```

Comment:
Do you want an actual email here, or are you trying to avoid spam? (Unless this is your actual email, in which case ignore me)

Reply:
I'll update it with my email, that's a remnant of a template that I pulled from